src/parse-stream.js

```diff
@@ -463,4 +469,23 @@ export default class ParseStream extends Stream {
       }
     });
   }
+
+  /**
+   * Add a custom header mapper
+   *
+   * @param {Object}   options
+   * @param {RegExp}   options.expression   a regular expression to match the custom header
+   * @param {Function} options.map          function to translate tag into a different tag
+   */
+  addTagMapper({expression, map}) {
+    const mapFn = line => {
```

Comment:
Looks like the example in the docs repeats the regex test(by executing the regex) and it doesn't need to? Maybe we could actually do a match here and pass it into the map option so the regex can be tested once?

Reply:
The doc example is testing that the custom tag has a value for it to pass along to the new tag, not that the tag. I don't think we should pass along the regex since it's not testing the same thing.